components/Pages/CarePackages/BrokerCareCharge/PreviousCareCharge.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Announcement, Button, Container, HorizontalSeparator, VerticalSeparator } from '../../../HackneyDS';
+
+export const PreviousCareCharges = ({ usePreviousCareCharge, useNewCareCharge }) => (
+  <Announcement
+    className='warning'
```

Comment:
use the isError prop as this was designed

Reply:
this is not error announcement
https://www.figma.com/file/Jc6Pxo05rghhH47fcDt9mm/Finance?node-id=7899%3A104010

Reply:
So then remove the class altogether?

Reply:
And it as a prop

Reply:
done